File: executable/julia/error-bar-horizontal.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "x" => [1, 2, 3, 4], 
    "y" => [2, 1, 3, 4], 
    "error_x" => [
      "value" => 10, 
      "type" => "percent"
    ], 
    "type" => "scatter"
  ]
]

response = Plotly.plot([data], ["filename" => "error-bar-horizontal", "fileopt" => "overwrite"])
plot_url = response["url"]

File: executable/python/error-bar-horizontal.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

data = Data([
    Scatter(
        x=[1, 2, 3, 4],
        y=[2, 1, 3, 4],
        error_x=ErrorX(
            value=10,
            type='percent'
        )
    )
])

plot_url = py.plot(data, filename='error-bar-horizontal')